xml/System.Linq.Expressions/Expression.xml

```diff
@@ -10656,19 +10656,19 @@ As with `Func`, the last argument is the return type. It can be set to `System.V
 
  One or more elements in <paramref name="parameters" /> are <see langword="null" />.</exception>
         <exception cref="T:System.ArgumentException">
-          <paramref name="TDelegate" /> is not a delegate type.
+          <typeparamref name="TDelegate" /> /> is not a delegate type.
```

Comment:
Is the second `/>` in `<typeparamref name="TDelegate" /> />` expected? Same question for cases below.

Reply:
Oops, thanks.